modules/eplanningBidAdapter.js

```diff
@@ -36,76 +36,19 @@ export const spec = {
   },
 
   buildRequests: function(bidRequests, bidderRequest) {
```

Comment:
Here we have changed return type from object to array of object. Is it compatible?

Reply:
I have checked showing no errors
Here is the screenshot
![image](https://github.com/adpushup/Prebid.js/assets/126073485/a619236d-3996-4184-bb90-aede82803f23)
![image](https://github.com/adpushup/Prebid.js/assets/126073485/2c913707-14ad-4621-9381-ee787e7bb445)

and same impletions is in rubicon bidadapter.